node/config.go

```diff
@@ -33,7 +33,7 @@ var (
 		0: "eth_quorum",
 		1: "permissioned_quorum",
```

Comment:
Not sure how this is used. Do we need to update this to `eigen_quorum`?

Reply:
+1